Calibration/LumiAlCaRecoProducers/plugins/AlcaPCCProducer.cc

```diff
@@ -88,11 +88,11 @@ void AlcaPCCProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
         DetId detId = mod.id();
 
         // -- clusters on this det
-        edmNew::DetSet<SiPixelCluster>::const_iterator  di;
-        int nClusterCount=0;
-        for (di = mod.begin(); di != mod.end(); ++di) {
-            nClusterCount++;
-        }
+        //edmNew::DetSet<SiPixelCluster>::const_iterator  di;
+        //int nClusterCount=0;
+        //for (di = mod.begin(); di != mod.end(); ++di) {
+        //    nClusterCount++;
+        //}
```

Comment:
We plan to use this later when we add a threshold for the pixels. Will place an explanation in comment and then push again.